cmd/utils/flags.go

```diff
@@ -491,6 +493,12 @@ var (
 		Usage:    "Enable recording the SHA3/keccak preimages of trie keys",
 		Category: flags.PerfCategory,
 	}
+	CacheLogCacheSizeFlag = &cli.IntFlag{
```

Comment:
nit: isnt this a bit repetitive? cache log cache

Reply:
Yeah it's not the best name, I agree. `--cache.log` is too short. Maybe `--cache.blocklogs`?